packages/search-in-workspace/src/node/ripgrep-search-in-workspace-server.ts

```diff
@@ -82,10 +94,15 @@ export class RipgrepSearchInWorkspaceServer implements SearchInWorkspaceServer {
     }
 
     protected getArgs(options?: SearchInWorkspaceOptions): string[] {
-        const args = ['--json', '--max-count=100'];
+        const args = ['--hidden', '--json'];
         args.push(options && options.matchCase ? '--case-sensitive' : '--ignore-case');
         if (options && options.includeIgnored) {
-            args.push('-uu');
+            args.push('--no-ignore');
+        }
+        if (options && options.maxFileSize) {
+            args.push('--max-filesize=' + options.maxFileSize.trim());
+        } else {
+            args.push('--max-filesize=20M');
```

Comment:
it would be really nice if these changes could be described in the commit message

Reply:
Added a bullet point to the commit message.